code/modules/integrated_electronics/subtypes/output.dm

```diff
@@ -204,6 +204,15 @@
 		)
 	spawn_flags = IC_SPAWN_RESEARCH
 
+/obj/item/integrated_circuit/output/sound/vox
+	name = "ai vox sound circuit"
+	desc = "Takes a sound name as an input, and will play said sound when pulsed. This circuit is often found in AI announcement systems."
+	spawn_flags = IC_SPAWN_RESEARCH
+
+/obj/item/integrated_circuit/output/sound/vox/Initialize()
+	.= ..()
+	sounds = GLOB.vox_sounds
+	extended_desc = "The first input pin determines which sound is used. It uses the AI Vox Broadcast word list. So either experiment to find words that work, or ask the AI to help in figuring them out. The second pin determines the volume of sound that is played, and the third determines if the frequency of the sound will vary with each activation."
```

Comment:
Now this could be moved to type definition.

Reply:
can't, Initialize proc overrides it

Reply:
:confused: